client/src/main/java/io/pravega/client/batch/impl/BatchClientFactoryImpl.java

```diff
@@ -220,6 +225,21 @@ public List<SegmentRange> getSegmentRangeBetweenStreamCuts(final StreamCut start
     public StreamCut getNextStreamCut(final StreamCut startingStreamCut, long approxDistanceToNextOffset) throws SegmentTruncatedException {
         log.debug("getNextStreamCut() -> startingStreamCut = {}, approxDistanceToNextOffset = {}", startingStreamCut, approxDistanceToNextOffset);
         Preconditions.checkArgument(approxDistanceToNextOffset > 0, "Ensure approxDistanceToNextOffset must be greater than 0");
+        return Retry.withExpBackoff(1, 10, 4, Duration.ofSeconds(60).toMillis()).retryWhen(t -> {
```

Comment:
Can we increase no of retries here?

Reply:
made number of retries same (10) as used at other places in client package